packages/api/src/configuration/constants.ts

```diff
@@ -18,3 +18,12 @@
 
 export const CONFIGURATION_DEFAULT_SCOPE = 'DEFAULT';
 export const CONFIGURATION_ONBOARDING_SCOPE = 'Onboarding';
+export const CONFIGURATION_MANAGED_DEFAULTS_SCOPE = 'MANAGED_DEFAULTS';
+
+// Below is the location of the managed defaults file on different OS'
+// this will be the Managed by profile which will show some "suggested defaults" by
+// the system administrator.
+export const MANAGED_DEFAULTS_FILE_MAC = '/Library/Application Support/com.podman.desktop/default-settings.json';
+export const MANAGED_DEFAULTS_FILE_WINDOWS_DIR = 'PodmanDesktop';
+export const MANAGED_DEFAULTS_FILE_WINDOWS_FILE = 'default-settings.json';
+export const MANAGED_DEFAULTS_FILE_LINUX = '/usr/share/podman-desktop/default-settings.json';
```

Comment:
IMHO MANAGED is not helpful there

should only be DEFAULTS_* (in the separate file/class)